types/ember-data/index.d.ts

```diff
@@ -1199,15 +1122,13 @@ export namespace DS {
                 backgroundReload?: boolean | undefined;
                 include?: string | undefined;
                 adapterOptions?: any;
-            }
+            },
         ): PromiseArray<ModelRegistry[K], Ember.ArrayProxy<ModelRegistry[K]>>;
         /**
          * This method returns a filtered array that contains all of the
          * known records for a given type in the store.
          */
-        peekAll<K extends keyof ModelRegistry>(
-            modelName: K
-        ): RecordArray<ModelRegistry[K]>;
+        peekAll<K extends keyof ModelRegistry>(modelName: K): RecordArray<ModelRegistry[K]>;
```

Comment:
@gitKrystan @chriskrycho

Since ember-data@4.7, `The return values of hasMany relationships, peekAll, findAll and query are now proxies to native arrays` 

Would it be possible to update this DefinitelyTyped package to reflect this change?